spec/ndx-ibl-metadata.extensions.yaml

```diff
@@ -60,83 +60,42 @@ groups:
   neurodata_type_inc: Subject
   name: subject
   doc: IBL mice data
-  attributes:
-  - name: alive
-    dtype: bool
-    doc: alive/dead
-    required: false
   datasets:
-  - name: nickname
+  - name: uuid
     dtype: text
-    doc: name of the mouse
+    doc: The full identifier of the subject from the IBL database.
```

Comment:
I'm proposing here that we set the `subject_id` in the NWB file to the `nickname` field of each subject, because that seems to be the primary value that is interlinked even within their database to the lab and session information

Also because the `nickname` is more human-readable, and will show up in the filename on the dandi set

But it's also not a bad idea to keep the full UUID attached to the subject for provenance